cirq-ionq/cirq_ionq/service_test.py

```diff
@@ -296,3 +296,28 @@ def test_service_remote_host_default():
 def test_service_remote_host_from_env_var_cirq_ionq_precedence():
     service = ionq.Service(api_key='tomyheart')
     assert service.remote_host == 'http://example.com'
+
+
+def test_service_run_unwraps_single_result_list():
```

Comment:
can you add a test for the correctness of the order of multiple results being returned?